testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,77 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev python3-dev freeglut-dev glew-dev lcms2-dev tinyxml-dev yaml-cpp-dev boost-dev"
+install=""
+subpackages="py2-$pkgname:py2 py3-$pkgname:py3 $pkgname-dev $pkgname-utils"
```

Comment:
It’d be better to rename opencolorio-utils to opencolorio-tools. This name uses Fedora and Debian, so it’d be more clear for users coming from main-stream distros.